.eslintrc.yml

```diff
@@ -0,0 +1,27 @@
+extends:
+  - defaults/configurations/airbnb/es5
+rules:
+  no-empty-label: off
+  space-after-keywords: off
+  space-return-throw-case: off
+  no-labels: error
+  keyword-spacing: error
```

Comment:
Alternatively, I might try to switch to using `eslint-config-airbnb-base/legacy` directly from https://github.com/airbnb/javascript.